README.md

```diff
@@ -25,3 +30,16 @@ See the [list of image tags available on Docker Hub|https://hub.docker.com/r/met
 ## Building and deploying images
 
 Use [`push.sh`](push.sh) script.  The way it operates is detailed in its heading comments.
+
+## Available images
+
+We have these images available:
+
+image version | python version | consul-template version
+----|----|----
+2.7, 2.7-20201201 | 2.7.18 | 0.16
+3.6-1 | 3.6.8 | 0.18
+3.6, 3.6-20201201 | 3.6.12 | 0.16
+3.7, 3.7-20201201 | 3.7.9 | 0.16
+3.8, 3.8-20201201 | 3.8.6 | 0.16
+3.9, 3.9-20210115 | 3.9.1 | 0.18
```

Comment:
15th of January is tomorrow, I guess you plan to build them on tomorrow then, right?

Reply:
yes :) I was just anticipating when I will build it.